docs/build/tooling/hardhat/migrating-to-zksync.md

```diff
@@ -214,7 +220,7 @@ export default async function (hre: HardhatRuntimeEnvironment) {
 ```
 
 ::: tip Test ETH
-Obtain [test ETH from third party faucets](../../support/faq.md#how-do-i-request-funds-for-testnet) or just bridge GöerliETH using [bridges](https://zksync.io/explore#bridges).
+Obtain [test ETH from third party faucets](../../support/faq.md#how-do-i-request-funds-for-testnet) or just bridge Sepolia ETH using [bridges](https://zksync.io/explore#bridges).
```

Comment:
We should link to the `build/tooling/network-faucets.md` file instead of the FAQ for third party faucets